.github/workflows/integration-test.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - name: Add mock connectors
         working-directory: tests
         run: |
-          npm run cli connector add @logto/connector-mock-sms @logto/connector-mock-email @logto/connector-mock-social -- -p ../logto
+          npm run cli connector add @logto/connector-mock-sms @logto/connector-mock-email @logto/connector-mock-standard-email @logto/connector-mock-social -- -p ../logto
```

Comment:
could we use the new standard connectors to replace the original mock connectors?  Since they all support multi instances after this refactor

Reply:
Yes. We can do this later in the cooldown cycle, as the change you mentioned is out of scope. Add LOG-4862 and LOG-4863 to track this change.